android-project/app/src/main/java/oceantreasur/es/fragments/ActivityMain.java

```diff
@@ -1,144 +1,152 @@
 package oceantreasur.es.fragments;
 
-import android.support.v4.app.FragmentManager;
-import android.support.v4.app.FragmentTransaction;
-import android.support.v4.view.ViewPager;
+
+import android.app.FragmentManager;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.support.v7.app.AlertDialog;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.util.Log;
 import android.view.LayoutInflater;
 import android.view.View;
-import android.view.animation.Animation;
-import android.view.animation.TranslateAnimation;
+import android.view.Window;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
 import android.widget.RelativeLayout;
 
-import java.util.ArrayList;
-import java.util.Random;
+import java.net.SocketTimeoutException;
 
 import oceantreasur.es.R;
-
-import static oceantreasur.es.R.dimen.small_fish_height;
-import static oceantreasur.es.R.dimen.small_fish_width;
-import static oceantreasur.es.R.layout.big_fish;
-import static oceantreasur.es.R.layout.small_fish;
-import static oceantreasur.es.view.AnimationConstants.*;
-import static oceantreasur.es.view.ScreenUtils.*;
+import oceantreasur.es.network.OceanTreasuresApplication;
+import oceantreasur.es.network.model.CheckAnswerRequest;
+import oceantreasur.es.network.model.CheckAnswerResponse;
+import oceantreasur.es.network.model.NextWordResponse;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
 
 public class ActivityMain extends AppCompatActivity {
 
-    android.app.FragmentManager fragmentManager;
+    android.app.FragmentManager fragmentManagaer;
     android.app.FragmentTransaction fragmentTransaction;
-    private ArrayList<View> smallFishViews = new ArrayList<>();
-    private ArrayList<View> bigFishViews = new ArrayList<>();
-    private int smallFishBeginOffset;
-    private int bigFishBeginOffset;
+    private NextWordResponse nextWord;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main_fragments);
 
-        smallFishBeginOffset = getDimensionOfFishInPx(small_fish_width);
-        bigFishBeginOffset = getDimensionOfFishInPx(R.dimen.big_fish_width);
+        fragmentManagaer = getFragmentManager();
 
-        fragmentManager= getFragmentManager();
-        fragmentTransaction = fragmentManager.beginTransaction();
+        RelativeLayout background = (RelativeLayout) findViewById(R.id.fish_background);
+        AnimationController animationController = new AnimationController(background);
```

Comment:
Bad architecture. The actions, performed by this class should not be invoked in the constructor of the class.